challenge.md

```diff
@@ -32,19 +32,19 @@ Open up this file in your text editor, and write your answers below (where indic
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
- 
+ It is a relative path because of the /
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
  <!-- Answer Starts Here -->
- 
+ An absolute path becuase we want to refer to the location of the file and directories within the file system.
```

Comment:
It is an absolute path because it is a URL, and a URL is designed to be an Absolute path. Why? Because people in different locations (say, my browser and your browser) need to be able to use the same path to access the same resource (such as a web page).